CMakeLists.txt

```diff
@@ -8,9 +8,14 @@ cmake_policy(SET CMP0069 NEW)
 # Honor visibility properties for all targets
 # Set the default so subdirectory cmake_minimum_required calls won't unset the policy.
 cmake_policy(SET CMP0063 NEW)
+set(CMAKE_POLICY_DEFAULT_CMP0063 NEW)
```

Comment:
It was already there near the bottom before I made any changes (line 13 on the original); I just reorganized it and placed it with the similar one above it because of my ocd.

Reply:
Oh I see mb